src/build/patches.ts

```diff
@@ -63,6 +63,20 @@ function handleTyped(type: Node): Typed {
   };
 }
 
+function handleAdditionalTypes(node: Node) {
+  const additionalTypes = [];
+  for (const child of node.children) {
+    if (child.name === "additionalType") {
```

Comment:
This is assuming to repeat like

```
additionalType foo
additionalType bar
```

right? But maybe just `additionalTypes foo bar`, because it's always just a simple string?